dashboard/app/models/regional_partner.rb

```diff
@@ -15,6 +15,14 @@
 
 class RegionalPartner < ActiveRecord::Base
   belongs_to :contact, class_name: 'User'
+
+  has_many :regional_partner_organizers
+  has_many :organizers,
+    class_name: 'User',
+    through: :regional_partner_organizers
```

Comment:
I don't think `organizers` is the best name for this association. They're only organizers in relation to workshops, but in the context of the regional partners it may not be the best label. I checked w Carina and she suggested `program_managers`. What do you think? We could also go with something shorter or just `users`.